src/lambda/commands/deploySamApplication.ts

```diff
@@ -26,19 +33,34 @@ export async function deploySamApplication(
 
     const tempFolder = await mkdtemp('samDeploy')
     const outputTemplatePath = path.join(tempFolder, 'template.yaml')
-
+    let stage = 'packaging'
     try {
         const packageInvocation = new SamCliPackageInvocation(template.fsPath, outputTemplatePath, s3Bucket, invoker)
+        restParams.outputChannel.show(true)
+        // TODO: Add nls support
+        restParams.outputChannel.appendLine(`Packaging SAM app to "${s3Bucket}" S3 bucket`)
         await packageInvocation.execute()
 
+        stage = 'deploying'
         const deployInvocation = new SamCliDeployInvocation(outputTemplatePath, stackName, invoker)
+        // Deploying can take a very long time for Python Lambda's with native dependencies so user needs feedback
+        restParams.outputChannel.appendLine(localize(
+          'AWS.samcli.deploy.stackName.initiated',
+          'Deploying "{0}" stack...',
```

Comment:
Minor: Consider: 
`Deploying SAM Application to CloudFormation Stack: {0}`

Reply:
✔